kong/runloop/handler.lua

```diff
@@ -1090,8 +1090,8 @@ return {
     after = function(ctx)
       local ok, err, errcode = balancer_execute(ctx)
       if not ok then
-        local body = utils.get_default_exit_body(errcode, err)
-        return kong.response.exit(errcode, body)
+        local body = utils.get_default_exit_message(errcode, err)
+        return kong.response.error(errcode, body)
```

Comment:
Do we even need to get the default exit message? Isn't the response.error doing it automatically?

Reply:
It does, there is a difference though: `get_default_exit_message` have [message override mechanism](https://github.com/Kong/kong/blob/c8d370475197c2f9585068b930cb84b7db11c133/kong/tools/utils.lua#L1120) that `response.error` do not have.... That's why I let it like that...